### PR TITLE
New methods for rpa functions

### PR DESCRIPTION
**Pull request by UzielLinares**
This PR add new methods for the rpa functions in order to support `TaylorModel1`'s with `TaylorN` coefficients.

### CODE REVIEW

**Comment by lbenet** [comment]
Is this ready for review?

**Comment by UzielLinares** [comment]
I think this is ready for revision.

File: src/bounds.jl [hunk 1]
```diff
@@ -34,6 +34,28 @@ function bound_remainder(::Type{TaylorModel1}, f::Function, polf::Taylor1, polfI
     end
     return Δ
 end
+function bound_remainder(::Type{TaylorModel1}, f::Function, polf::Taylor1{TaylorN{T}}, polfI::Taylor1, x0, I::Interval) where {T}
```
Comment (lbenet) [inline src/bounds.jl:37]: Could you add some comment to make explicit that we are assuming that the domain of the `TaylorN` part is the  normalized symmetric box?

**Review by lbenet — COMMENTED** [review]
Thanks for your contribution. I have added a simple suggestion to include a comment that the domain of the `TaylorN` part is assumed to be  the normalized box; memory is a  finite thing, and we are very prone to forget! Can you add it?

Aside from that, LGTM!

File: src/bounds.jl (hunk 1 above)
Comment (UzielLinares) [reply src/bounds.jl:37]: ready!

**Review by lbenet — APPROVED** [review]
Thanks a lot!

**Comment by lbenet** [comment]
Merging...

**Pull request merged**